.github/workflows/ci.yml

```
@@ -42,7 +42,7 @@ jobs:
           distribution: adopt
           cache: 'sbt'
 
-      - uses: sbt/setup-sbt@v1
+      - uses: sbt/setup-sbt@v1.1
```

```suggestion
      - uses: sbt/setup-sbt@v1
```

Sorry, it seems I was wrong when I suggested this change. GitHub actions seem to supports only a major version or a full version, but not a minor version…